packages/rum-core/src/boot/rumPublicApi.ts

```diff
@@ -157,6 +157,8 @@ export function makeRumPublicApi(
         hasReplay: recorderApi.isRecording() ? true : undefined,
       }),
       recorderApi,
+      globalContextManager,
+      userContextManager,
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏if startRum has now access to all that, `getCommonContext` could be declared there instead of here

Reply:
I don't really get the design you have in mind but I thought that we could remove the `getCommonContext` once the `contextManagers` will use context history.

Reply:
I tryied to do as less change as possible but due to this [test](https://github.com/DataDog/browser-sdk/blob/90eee94c1ec89a208a423a4ac59c912a119ad30b/packages/rum-core/src/boot/rumPublicApi.spec.ts#L830), I moved `commonContext` logic to /context. 
Like I suspected, it polluted a bit this PR. Let me know what you think